.github/CODEOWNERS

```diff
@@ -0,0 +1,34 @@
+# See docs: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+
+# Default unless a later match takes precedence
+*       @reubenmiller @didier-wenzek @albinsuresh
+
+# Build
+justfile @thin-edge/contributors @thin-edge/inetum-contributors
+/.devcontainer/ @reubenmiller @albinsuresh
+
+# CI
+/.github/ @reubenmiller @Bravo555 @rina23q
+
+# Documentation
+/docs/ @reubenmiller @didier-wenzek
+/design/ @reubenmiller @didier-wenzek @albinsuresh
+/images/ @reubenmiller @didier-wenzek
+
+# Testing
+/tests/ @reubenmiller @gligorisaev
```

Comment:
I'm okay with that, as this really makes sense. However, this doesn't match our current practice. With this enforced, most of the PRs will have to be reviewed by @gligorisaev or @reubenmiller, because containing system tests.

Reply:
Resolved by 887959186e57beecaee6274fc23e04ae8de2ff75